clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5150,9 +5150,11 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
   const bool IsAssertBuild = true;
 #endif
 
-  // Disable the verification pass in -asserts builds.
```

Comment:
Hmm, I might be missing something here, but doesn't this disable the verifier for builds without assertions?

Reply:
I edited the line to append the `unless otherwise specified`, since now we allow overriding that with the new flag.